src/common/snippets/include/snippets/op/convert_saturation.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/op/convert.hpp>
+#include "ngraph/op/op.hpp"
+
+namespace ngraph {
+namespace snippets {
+namespace op {
+
+/**
+ * @interface ConvertSaturation
+ * @brief The implementation uses "saturation" conversion.
```

Comment:
We created these separate Converts because we had problems with integer saturation. So these Converts don't have specific rules for floating-points values. I added notes that these Converts are ordinary but with specific rules for integer conversion